core/primitives/src/version.rs

```diff
@@ -136,12 +138,24 @@ pub enum ProtocolFeature {
 /// Current latest stable version of the protocol.
 /// Some features (e. g. FixStorageUsage) require that there is at least one epoch with exactly
 /// the corresponding version
-#[cfg(not(feature = "nightly_protocol"))]
+#[cfg(all(
+    not(feature = "nightly_protocol"),
+    any(
+        not(feature = "protocol_feature_new_validator_selection_algorithm"),
+        feature = "protocol_feature_chunk_only_producers"
+    )
+))]
```

Comment:
why?

Reply:
We need a way to test with just protocol_feature_new_validator_selection_algorithm enabled

Reply:
Also see my comment about that in description